packages/frontend/src/types/table-core.d.ts

```diff
@@ -1,10 +1,9 @@
 import { Field, PivotReference, TableCalculation } from '@lightdash/common';
-import { RowData } from '@tanstack/react-table';
 import { MouseEventHandler } from 'react';
 import { Sort } from '../components/common/Table/types';
 
 declare module '@tanstack/table-core' {
-    interface ColumnMeta<TData extends RowData, TValue> {
+    interface ColumnMeta {
```

Comment:
not sure about this change, why was this removed here? 🤔

Reply:
unused types